contracts/merger/MergerBase.sol

```diff
@@ -0,0 +1,66 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "../dao/Timelock.sol";
+
+/// @title Base contract for merger logic
+/// @author elee
+contract MergerBase {   
+    event Accept(address indexed dao);
+    event Enabled(); 
+
+    uint256 public constant scalar = 1e9;
```

Comment:
Both tokens have 18 decimals so scalar should be 1e18, not 1e19.

Reply:
its a custom scalar for the exchange rate, not the tokens